pkgs/applications/audio/parlatype/default.nix

```
@@ -0,0 +1,69 @@
+{ stdenv, fetchFromGitHub, pkgconfig, meson, gnome3, at-spi2-core, dbus, gst_all_1, sphinxbase, pocketsphinx, ninja, gettext, appstream-glib, python3, glib, gobject-introspection, gsettings-desktop-schemas, itstool, wrapGAppsHook, makeWrapper, hicolor-icon-theme }:
+
+stdenv.mkDerivation rec {
+  name = "parlatype-${version}";
+  version = "v1.6-beta";
```

```suggestion
  version = "1.6-beta";
```

It is not typical to add the 'v' prefix to a version.